src/hotspot/share/opto/vectorization.hpp

```diff
@@ -1320,4 +1323,11 @@ class AlignmentSolver {
 #endif
 };
 
+struct VTransformBoolTest {
+  const BoolTest::mask _mask;
+  const bool _is_negated;
+  VTransformBoolTest(const BoolTest::mask mask, bool is_negated) :
```

Comment:
Maybe add a new line to better separate from the constructor:
```suggestion

  VTransformBoolTest(const BoolTest::mask mask, bool is_negated) :
```